CHANGELOG.md

```diff
@@ -18,6 +18,7 @@ Changes since last non-beta release.
 *Please add entries here for your pull requests that are not yet released.*
 - Added `./bin/dev` and `./bin/dev-static` executables to ease and standardize running the dev server. [PR 1491](https://github.com/shakacode/react_on_rails/pull/1491) by [ahangarha](https://github.com/ahangarha)
```

Comment:
1. Let's make this 13.3.0
2. Add a `Fixed` and `Added` sections. See blow, line 24 and 28.

Reply:
Done! Added a commit